eclipse-scout-core/src/form/fields/FormFieldValidationResultProvider.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2010, 2024 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+import {fields, FormField, InitModelOf, ObjectModel, ObjectWithType, SomeRequired, Status, ValidationResult} from '../..';
+
+/*
+ * Copyright (c) 2010, 2024 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+export class FormFieldValidationResultProvider implements ObjectWithType {
```

Comment:
It does actually not validate but only create the validation result based on the empty and error status properties. Because of that and because of the naming conflict with the actual validators of the value field, I would rather not name it validator.